Index: arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts b/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
--- a/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts	(revision 29eccf4b37700ba77678acd0fb2fc330def05286)
+++ b/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts	(revision 9134a321eff412667c0828ff803e43c49e2f08e3)
@@ -309,9 +309,9 @@
 	#size-cells = <0>;
 
 	bus-width = <4>;
-	cap-sd-highspeed;
-	sd-uhs-sdr50;
-	max-frequency = <100000000>;
+	/* cap-sd-highspeed; */ /* SDIO: Completely remove any highspeed capability */
+	/* sd-uhs-sdr50; */ /* SDIO: Completely remove any highspeed capability */
+	max-frequency = <80000000>; /* SDIO: Reduce speed */
 
 	non-removable;
 	disable-wp;
@@ -338,8 +338,8 @@
 	pinctrl-names = "default", "clk-gate";
 
 	bus-width = <4>;
-	cap-sd-highspeed;
-	max-frequency = <100000000>;
+	/* cap-sd-highspeed; */ /* SD: Completely remove highspeed capability */
+	max-frequency = <80000000>; /* SD: Reduce speed */
 	disable-wp;
 
 	cd-gpios = <&gpio GPIOC_6 GPIO_ACTIVE_LOW>;
